backend/local/local.go

```diff
@@ -1374,7 +1374,7 @@ type localChunkWriter struct {
 }
 
 func (w *localChunkWriter) WriteChunk(chunkNumber int, reader []byte) (int, error) {
-	offset := int64(chunkNumber) * w.chunkSize
+	offset := int64(chunkNumber-1) * w.chunkSize
```

Comment:
We should probably start the chunks from 0 otherwise we'll be confusing all the 0-based Go programmers until the end of time ;-)

Reply:
That's probably a good idea 😅 
Sorry, I was thinking on the S3 API while implementing this (where part numbers start with 1).
I updated the code to start chunks from 0.